examples/12.Misc/Stereo/Stereo.ino

```diff
@@ -12,7 +14,12 @@
  *  This example code is in the public domain.
  */
 
-#include <MozziGuts.h>
+// Configure Mozzi for Stereo output. This must be done before #include <Mozzi.h>
+// MozziConfigValues.h provides named defines for available config options.
+#include <MozziConfigValues.h>
+#define MOZZI_AUDIO_CHANNELS MOZZI_STEREO
```

Comment:
Wonder if (that might already be the case), `#define MOZZI_AUDIO_CHANNELS 2` also work?

Reply:
Indeed it would, MOZZI_STEREO is defined as 2.